### ISSUE TITLE
enable configurable logging level via configuration

### ISSUE THREAD

**Issue by gilv** [pywren/pywren-ibm-cloud#55]
Current code contains statements like 
wrenlogging.ow_config(logging.DEBUG)
level = logging.DEBUG ( jobrunner )

We need to extend configuration and control in runtime the logging level we want to have. This configuration will be used as a logging level for all PyWren components

**Comment by JosepSampe** [comment]
Instead of configuration, wich is fixed for all pywren executions, IBM-PyWren already contains a parameter to set internal log level:

`pw = pywren.ibm_cf_executor(log_level='DEBUG')`

However, code executed in CF does not get this configuration. It should be propagated.

**Comment by gilv** [comment]
The default if log_level is not provided should be INFO (to reduce number of log prints )

**Comment by gilv** [comment]
Adding more to this thread, logging level should get to cos.py, that currently contains

    logging.getLogger('ibm_boto3').setLevel(logging.CRITICAL)
    logging.getLogger('ibm_botocore').setLevel(logging.CRITICAL)
    logging.getLogger('urllib3').setLevel(logging.CRITICAL) 
    logger = logging.getLogger(__name__)

**Comment by gilv** [comment]
@omerb01 can you please handle this?

**Comment by gilv** [comment]
@JosepSampe we can provide log level as you suggested 

     pw = pywren.ibm_cf_executor(log_level='DEBUG')

but this one also works, what is the difference, if any?

     import logging
     logging.basicConfig(level=logging.DEBUG)
     pw = pywren.ibm_cf_executor()

**Comment by JosepSampe** [comment]
@gilv They are two different ways to set logging level.
The one integrated in PyWren (`pw = pywren.ibm_cf_executor(log_level='DEBUG')`) uses the wrenlogging.py module, that uses a personalized log format:
```
2019-03-20 08:51:08,083 [INFO] pywren_ibm_cloud.cf_connector: IBM Cloud Functions init for namespace: josep.sampe@urv.cat_us_east
2019-03-20 08:51:08,083 [INFO] pywren_ibm_cloud.cf_connector: IBM Cloud Functions init for host: https://us-east.functions.cloud.ibm.com
2019-03-20 08:51:08,083 [INFO] pywren_ibm_cloud.invokers: IBM Cloud Functions init for Runtime: pywren_3.6
2019-03-20 08:51:08,626 [INFO] pywren_ibm_cloud.executor: IBM Cloud Functions executor created with ID 8425f7a3-fed7
```

In contrast, the other case you need to import the logging  module and manually set the logging.basicConfig, which has another log format:
```
INFO:pywren_ibm_cloud.cf_connector:IBM Cloud Functions init for namespace: josep.sampe@urv.cat_us_east
INFO:pywren_ibm_cloud.cf_connector:IBM Cloud Functions init for host: https://us-east.functions.cloud.ibm.com
INFO:pywren_ibm_cloud.invokers:IBM Cloud Functions init for Runtime: pywren_3.6
INFO:pywren_ibm_cloud.executor:IBM Cloud Functions executor created with ID 484b1049-4eb3
```
Maybe from a user perspective it is easier to set the log level parameter in the executor rather than have to import the logging package and manually create the logging config in each file.

**Comment by omerb01** [comment]
@gilv logging level gets already to cos.py
do you want to get information from packages `ibm_boto3`, `ibm_botocore` and `urllib3` also? cause currently for these packages, logging level is set to CRITICAL only.

**Comment by gilv** [comment]
@omerb01 this is cos.py looks like

logging.getLogger('ibm_boto3').setLevel(logging.DEBUG)
logging.getLogger('ibm_botocore').setLevel(logging.CRITICAL)
logging.getLogger('urllib3').setLevel(logging.CRITICAL)
logger = logging.getLogger(__name__)

logging doesn't get there.

@omerb01 let's folllow suggestion from @JosepSampe above.

**Comment by omerb01** [comment]
@gilv as we discussed, added an option to get `boto3` logs on pull request #100 

usage:
```python
pw = pywren.ibm_cf_executor(log_level='DEBUG_BOTO3')
```

**Comment by JosepSampe** [comment]
This issue is finally fixed in pull #102 
Now functions have same log_level as the local PyWren
